### ISSUE TITLE
How should I evaluate the performance of the network on my own test set ?

### ISSUE THREAD

**Issue by jjjenny1008** [qqwweee/keras-yolo3#419]
I have trained the model using my own images data set , but I want to evaluate the performance of my trained model on the test set , how should I do ? I'll appreciate it if someone could tell me.

**Comment by zh-plus** [comment]
#316 
This may help.

**Comment by canyilmaz90** [comment]
look at [this answer](https://github.com/qqwweee/keras-yolo3/issues/380#issuecomment-476444985)
